app/components/settings/billing-page/membership-section.ts

```diff
@@ -9,7 +10,11 @@ interface Signature {
   };
 }
 
-export default class MembershipSection extends Component<Signature> {}
+export default class MembershipSection extends Component<Signature> {
+  get isInstitutionMembership(): boolean {
```

Comment:
```suggestion
  get subscriptionSourceIsInstitutionMembershipGrant(): boolean {
```

(Our naming doesn't actually treat "InstitutionMembership" as a noun)